docs/source/user_guide/automl.ipynb

```diff
@@ -74,7 +74,7 @@
    "source": [
     "The AutoML search will log its progress, reporting each pipeline and parameter set evaluated during the search.\n",
     "\n",
-    "There are a number of mechanisms to control the AutoML search time. One way is to set the maximum number of candidate models to be evaluated during AutoML using `max_iterations`. By default, AutoML will search a fixed number of iterations and parameter pairs (`max_iterations=5`). The first pipeline to be evaluated will always be a baseline model representing a trivial solution. "
+    "There are a number of mechanisms to control the AutoML search time. One way is to set the `max_batches` parameter which controls the maximum number of rounds of AutoML to evaluate, where each round may train and score a variable number of pipeline. Another way is to set the `max_iterations` parameter which controls the maximum number of candidate models to be evaluated during AutoML. By default, AutoML will search for a single batch. The first pipeline to be evaluated will always be a baseline model representing a trivial solution. "
```

Comment:
Not related to this PR but when are we making `_pipelines_per_batch` public?

Reply:
@freddyaboulton yeah good q, no plans to do so currently, let's chat at standup